<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReimburseCheatClass</class>
 <widget class="QWidget" name="ReimburseCheatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>752</width>
    <height>674</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>752</width>
    <height>674</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReimburseCheat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget,#widget_3
{		
	background-color: rgb(255, 255, 255);
	
}

QWidget#widget_4
{	
	border-image: url(:/ReimburseCheat/Resources/1.png);	
	border: none;
	background: Transparent;
}

QLabel
{	
	font: 9pt &quot;宋体&quot;;	
	color: rgb(130, 130, 130);	
	background-color: rgb(255, 255, 255);
}

QLabel#label_4
{
	border-image: url(:/ReimburseCheat/Resources/minute.png);
	border: none;
	background: Transparent;
}

QLabel#lbStart,#lbEnd
{	
	font: 10pt &quot;宋体&quot;;	
	color: rgb(255, 255, 255);		
	background-color: rgb(128, 143, 158);
}

QLabel#lbScore
{		
	font: 75 26pt &quot;Arial&quot;;	
	color: rgb(255, 255, 255);			
	background-color: rgb(77, 155, 241);
}

QLabel#lbElapsed,#lbTop,#lbDistances,#lbAverage
{		
	font: 75 22pt &quot;Arial&quot;;	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8,6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,0,0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="edSig">
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="text">
               <string>4</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>格信号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edTime">
              <property name="text">
               <string>上午9:36</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edPower">
              <property name="text">
               <string>84</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <widget class="QPushButton" name="pushButton_3">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>170</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>取点</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>240</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>路线规划</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>240</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>截图保存</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>210</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>起点：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>240</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>终点：</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edStart">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>191</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>浙江省宁波市余姚市开丰路302...</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edEnd">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>191</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>浙江省宁波市余姚市保庆路269号</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edScore">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>20</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="edDistance">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>41</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>4.3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="edTop">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>70</y>
              <width>41</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>63</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="edAverage">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>70</y>
              <width>41</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>21</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="edElapsed">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>70</y>
              <width>41</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>12</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="edTip1">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>221</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>登录可得1枚金币(30分钟内有效)</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edTip2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>170</y>
              <width>221</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>准点到达，和高小德预计的时间相同哦</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="sLngLat">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>210</y>
              <width>141</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>121.135042,30.028951</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="eLngLat">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>240</y>
              <width>141</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>121.15658,30.043389</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>375</width>
           <height>447</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>402</y>
           <width>375</width>
           <height>270</height>
          </rect>
         </property>
         <widget class="QLabel" name="lbDistances">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>57</y>
            <width>65</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>4.3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbElapsed">
          <property name="geometry">
           <rect>
            <x>103</x>
            <y>57</y>
            <width>65</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>12</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbAverage">
          <property name="geometry">
           <rect>
            <x>196</x>
            <y>57</y>
            <width>65</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>21</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbTop">
          <property name="geometry">
           <rect>
            <x>293</x>
            <y>57</y>
            <width>65</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>63</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbTip1">
          <property name="geometry">
           <rect>
            <x>46</x>
            <y>134</y>
            <width>221</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>登录可得1枚金币(30分钟内有效)</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbTip2">
          <property name="geometry">
           <rect>
            <x>46</x>
            <y>189</y>
            <width>231</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>准点到达，和高小德预计的时间相同哦</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbStart">
          <property name="geometry">
           <rect>
            <x>33</x>
            <y>5</y>
            <width>195</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>浙江省宁波市余姚市开丰路302...</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="lbEnd">
          <property name="geometry">
           <rect>
            <x>33</x>
            <y>23</y>
            <width>195</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>浙江省宁波市余姚市保庆路269号</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="lbScore">
          <property name="geometry">
           <rect>
            <x>246</x>
            <y>9</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>100</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>109</x>
            <y>90</y>
            <width>56</width>
            <height>11</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ReimburseCheat.qrc"/>
 </resources>
 <connections/>
</ui>
